app/src/test/java/org/simple/clinic/sync/ModelSyncTest.kt

```diff
@@ -398,10 +412,16 @@ class ModelSyncTest {
               AppointmentSync(
                   syncCoordinator = syncCoordinator,
                   repository = mock(),
-                  api = mock(),
+                  apiV2 = mock(),
+                  apiV3 = mock(),
```

Comment:
Can you also add another instance of AppointmentSync where the v3 flag is enabled so that the code is exercised as a part of this test?